src/components/Rockets/Rocket.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './rockets.css';
+import { useDispatch } from 'react-redux';
+import { toggleReserve } from '../../Redux/Rockets/Rockets';
+
+const Rocket = (props) => {
+  const {
+    rocketId, rocketName, rocketDescription, rocketImage, rocketReserved,
+  } = props;
```

Comment:
- [ ] When the second rocket is not reserved it has some extra margin from the left side than other rockets. Please fix this issue and make sure all images are aligned with the same margins. Fixing this will improve the user interface of the application.

### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/174439369-68ced33c-bb6a-4414-b1ea-595fa25df2c2.png)

Reply:
Hi @thecodechaser ,
Thanks for the quick detailed review and clues to make the required changes. However, the screenshot provided to show us the required change on rocket margins appears to be different from what me and my partner have in our devices. So, we are not sure if there is an issue with it.

![image](https://user-images.githubusercontent.com/91727952/174444323-6afbef91-b08d-4938-986a-d84d84e8c353.png)

Reply:
@Christianib003 Check on different screens(more larger screens) if everything is fine from your end attach a screenshot to this PR
And when you go for another Review explain this situation to next CR in the code review request description form.